invenio_search_ui/config.py

```diff
@@ -55,3 +57,6 @@
 SEARCH_UI_JSTEMPLATE_SORT_ORDER = \
     'templates/invenio_search_ui/togglebutton.html'
 """Configure the toggle button template."""
+
+SEARCH_UI_SEARCH_CONFIG_GEN = SearchAppConfigHelper
```

Comment:
I would rename and make this a dictionary so that new configuration helpers can be added:

```suggestion
SEARCH_UI_SEARCH_CONFIG_HELPERS = {
    'invenio-records-rest': InvenioRecordsRESTSearchApp,
    # e.g. in the future:
    #   'invenio-resources': InvenioResourcesSearchApp,
}
```